frontend/src/pages/projects/components/ConnectedResourcesDeleteModal.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from 'react';
+import {
+  Badge,
+  Content,
+  ExpandableSection,
+  ExpandableSectionToggle,
+  Bullseye,
+  Spinner,
+} from '@patternfly/react-core';
+import { InferenceServiceKind, NotebookKind } from '#~/k8sTypes';
+import { getDisplayNameFromK8sResource } from '#~/concepts/k8s/utils';
+import { useNotebooksStates } from '#~/pages/projects/notebook/useNotebooksStates';
+
+type ConnectedResourcesDeleteModalProps = {
+  loaded: boolean;
+  connectedNotebooks: NotebookKind[];
+  connectedModels: InferenceServiceKind[];
+  namespace: string;
+};
+
+const ConnectedResourcesDeleteModal: React.FC<ConnectedResourcesDeleteModalProps> = ({
+  loaded,
+  connectedNotebooks,
+  connectedModels,
+  namespace,
+}) => {
+  const [modelsExpanded, setModelsExpanded] = React.useState<boolean>(false);
+  const [notebooksExpanded, setNotebooksExpanded] = React.useState<boolean>(false);
+  const [notebookStates] = useNotebooksStates(connectedNotebooks, namespace);
+
+  const getNotebookStatusText = React.useCallback(
+    (notebook: NotebookKind) =>
+      notebookStates.find((n) => n.notebook.metadata.name === notebook.metadata.name)?.isRunning
+        ? ' (Running)'
+        : '',
+    [notebookStates],
+  );
+  if (!loaded) {
+    return (
+      <Bullseye>
+        <Spinner />
+      </Bullseye>
+    );
+  }
+  return connectedNotebooks.length !== 0 || connectedModels.length !== 0 ? (
+    <>
+      <div className="pf-v6-u-mt-md">
+        {!connectedNotebooks.length && !connectedModels.length ? null : (
+          <>
+            {connectedNotebooks.length ? (
+              <>
+                <ExpandableSectionToggle
```

Comment:
It looks like line 45 and line 48 are basically doing the same checks? I think you can consolidate the ternary checks and have less nested tabs to make it easier to read
something like:
```suggestion
  if (connectionedNotebooks.length === 0 && connectedModels.length === 0) {
    return null
  }
  return (
    <div className="pf-v6-u-mt-md">
        {connectedNotebooks.length ? (
          <>
            <ExpandableSectionToggle
          ... : null
        }
        {connectedModels.length ? (
          <> 
            ....
```